docs/administration-guide/databases/google-analytics.md

```diff
@@ -2,49 +2,53 @@
+You will need to have a [Google Cloud Platform (GCP)][google-cloud] account and create the [project][google-cloud-create-project] you would like to use in Metabase. Consult the Google Cloud Platform documentation on how to [create and manage a project][google-cloud-management] if you do not have one.
 
 ## Google Cloud Platform: creating a service account and JSON file
 
-You'll first need a [service account](https://cloud.google.com/iam/docs/service-accounts) JSON file that Metabase can
+You'll first need a [service account][google-service-accounts] JSON file that Metabase can
 use to access your Google Analytics dataset. Service accounts are intended for non-human users (such as applications
 like Metabase) to authenticate (who am I?) and authorize (what can I do?) their API calls.
 
-To create the service account JSON file, follow Google's documentation on [setting up a service
-account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) for your Google Analytics dataset.
-Here's the basic flow:
-
-1. **Create service account**. From your Google Cloud Platform project console, open the main sidebar menu on the
-   left, go to the **IAM & Admin** section, and select **Service account**. The console will list existing service
-   accounts, if any. At the top of the screen, click on **+ CREATE SERVICE ACCOUNT**.
+To create the service account JSON file, follow Google's documentation on [setting up service accounts][google-managing-service-accounts] for your Google Analytics dataset.
 
-2. **Fill out the service account details**. Name the service account, and add a description (the service account ID
-   will populate once you add a name). Then click the **Create** button.
+1. From your [Google Cloud Platform console][google-cloud-platform-console], go to **IAM & Admin** > **Service accounts**.
 
-3. The **Service account permissions (optional)** section is not required. Click **Continue**.
+2. Click **+ CREATE SERVICE ACCOUNT** and fill out your service account details.
+   - Name the service account.
+   - Add a description (the service account ID will populate once you add a name). 
 
-4. **Create key**. Once you have assigned roles to the service account, click on the **Create Key** button, and select
-   **JSON** for the **key type**. The JSON file will download to your computer.
+3. Click **Continue** to skip the optional sections.
 
-> **You can only download the key once**. If you delete the key, you'll need to create another service account with
-> the same roles.
+4. Click **Done** to create your service account.
 
-### Adding the service account to your Google Analytics account
+4. From the three dot menu, go to **Manages keys** > **Add key**.
```

Comment:
```suggestion
4. From the **...** menu, go to **Manages keys** > **Add key**.
```
Nothing wrong with three dot, but we've been using **...**, so we might want to be consistent.